site/content/user-guide/amqp-adapter.md

```diff
@@ -65,12 +65,12 @@ The command-line client supports the following parameters (with default values):
 * `--message.payload`: The message payload body (default: `'{"temp": 5}'`)
 * `--hono.client.host`: The host name that the AMQP adapter is running on (default: `localhost`)
 * `--hono.client.port`: The port that the adapter is listening for incoming connections (default: `5672`)
+* `--hono.client.username`: The username to use for authenticating to the adpter (default: `sensor1@DEFAULT_TENANT`)
```

Comment:
please remove these, we do not use default values for credentials

Reply:
but we configured a default value for `username` and `password` in application.yml. can you please clarify?

Reply:
we have removed them from the yaml file ...

Reply:
ok 👍